### DISCUSSION TITLE
Welcome to PowerSchool-Student-Map Discussions!

### DISCUSSION THREAD

**Discussion by sriehl** [sriehl/PowerSchool-Student-Map#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by Wdickerson365** [comment]
Thank you for this wonderful plugin. Is there a way to include identifiers on the map for the school?

**Reply by sriehl** [reply]
I don't quite understand. It should have their student number when you click on the pin. Is there another identifier you were thinking of?

**Reply by Wdickerson365** [reply]
Appologies for not being clear. Is there a way to see the school iteslf on the map as a pin similar to the students? For example, if the school is on 123 Black Street, I could see it on the map alongside the student pins.

**Comment by miletiche** [comment]
That could be cool... If you are at a school, show that school, if you are
at the district office, show them all.

Thanks

Eric Miletich, CETL
IT Director
Kalispell Public Schools
(406) 751-3400 x3428
***@***.***



committees/memberships:
Montana Educational Technologists Association (NW Region Co-Chair)
<https://www.sammt.org/meta>
CoSN Professional Learning Committee (Co-Chair)
<https://www.cosn.org/committee/professional-advancement-committee/>
CoSN CTO Council
<https://www.cosn.org/committee/chief-technology-officer-council/>
PowerSchool Council on Education Innovation
<https://www.cosn.org/committee/professional-advancement-committee/>

**Comment by krichardson-cvsd** [comment]
Student Maps no longer shows in our PowerSchool after an update. Are are on PowerSchool Version: 24.12.0.0.
With there be an update to the Plugin?
